examples/vision/integrated_gradients.py

```diff
@@ -55,12 +55,15 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from scipy import ndimage
-from IPython.display import Image
+from IPython.display import Image, display
 
 import tensorflow as tf
-from tensorflow import keras
-from tensorflow.keras import layers
-from tensorflow.keras.applications import xception
+import keras
+from keras import layers
+from keras.applications import xception
+
+keras.config.disable_traceback_filtering()
```

Comment:
Likewise, you can remove this line in all 3 files

Reply:
Sorry, fixed.